test.mjs

```diff
@@ -27,5 +28,24 @@ function testContext({any, html, css, gql, md}, name) {
 			assert.equal(gql`a${'b'}c${1}`, 'abc1');
 			assert.equal(md`a${'b'}c${1}`, 'abc1');
 		});
+
+		it('strings with escapes', () => {
+			assert.equal(any`\\\na${'\\\na'}`, '\\\na\\\na');
```

Comment:
Why `\\\`

```suggestion
			assert.equal(any`\u{1F600}${'\u{1F600}'}\n${'\n'}`, '😀😀\n\n');
```

Reply:
Not `\\\` but `\\\n`, it's `\` +`line break`

Reply:
I added a combined emoji check.



> Prevent pass wrong raw to `String.raw`

I think `'\\'` already does that